packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -1364,28 +1449,59 @@ export function updateProperties(
           didWarnControlledToUncontrolled = true;
         }
       }
+
+      // Update checked *before* name.
+      // In the middle of an update, it is possible to have multiple checked.
+      // When a checked radio tries to change name, browser makes another radio's checked false.
+      if (
+        name != null &&
+        typeof name !== 'function' &&
+        typeof name !== 'symbol' &&
+        typeof name !== 'boolean'
+      ) {
+        if (__DEV__) {
+          checkAttributeStringCoercion(name, 'name');
+        }
+        domElement.setAttribute('name', name);
+      } else {
+        domElement.removeAttribute('name');
+      }
```

Comment:
setProp here too? and/or is it faster to assign to .name which should be equivalent?

Reply:
Same thing. I really just want to set `domElement.name = name` and I also want to make clear that `setProp` doesn't do anything complicated for this value.